verification/curator-service/ui/cypress/e2e/components/AppTest.spec.ts

```diff
@@ -6,43 +6,43 @@ describe('App', function () {
     beforeEach(() => {
         cy.task('clearSourcesDB', {});
         cy.seedLocation({
-            country: 'DEU',
+            country: 'DE',
```

Comment:
Are we going with three letter or two letter codes? I thought we had switched to three letters, could be mistaken. Either is fine as long as we use the same convention everywhere

Reply:
We do switch to ISO3, but the geolocation service is working with ISO2 codes. So I have to seed it with ISO2 codes.